File: OpenMP/colourless-1.f90
       program colourless
        implicit none 
	include "omp_lib.h"
	!$omp parallel
          print *, "Colourless " 
	  print *, "green "
	  print *, "ideas "
	  print *, "sleep furiously "
	!$omp end parallel
	  print *, ""
       end program colourless

File: OpenMP/colourless-1.c
#include <stdio.h>
#include <stdlib.h>
#include "omp.h"

int main (void)
{
#pragma omp parallel
 { 
	printf("Colourless ");
	printf("green ");
	printf("ideas ");
	printf("sleep furiously ");
   }
 printf("\n");
 return(0);
}	
